_test/test_mpi/test_job_dispatcher_herbert.m

```diff
@@ -13,6 +13,21 @@
             end
             this = this@job_dispatcher_common_tests(name,'herbert');
         end        
+        function test_job_fail_restart(obj, varargin)
+            if is_jenkins
+                return
```

Comment:
Do you think we should disable these tests outright rather than just not running them on Jenkins (e.g. prepend the test with `DISABLED_`? It feels cleaner and makes it obvious there's a problem. Likewise for running tests on Linux/Windows.

Reply:
I have enabled it this way, as my changes may affect this part of the test.

Reply:
And I want to know, why and where it running and where is not. 
As this runs on windows and linux on my machine but have problem on Jenkins, we know where to dig.
I think it should be carteful

Reply:
Additinally, there is separate problem on Windows. It mentioned in "disabled" text and should be addressed separately.